docs/creating_your_tooltip.md

```diff
@@ -65,8 +65,11 @@ Alternatively, you can create a runtime dataflow of the visualization using [`ve
 
 ```js
 var runtime = vega.parse(spec);
+var tooltip = new vegaTooltip.Handler(options);
```

Comment:
This snippet intentionally doesn't add the tooltips yet. Can you explain why you think it should add a tooltips?

Reply:
oh, perhaps I misunderstood that